pkgs/development/python-modules/bidict/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, buildPythonPackage, fetchPypi
+, setuptools_scm
+, sphinx
+, hypothesis
+, py
+, pytest
+, pytest-benchmark
+, sortedcollections
+, sortedcontainers
+}:
+
+buildPythonPackage rec {
+  pname = "bidict";
+  version = "0.17.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1icj0fnfx47n6i33pj5gfrmd1rzpvah1jihhdhqiqx2cy9rs6x4c";
+  };
+
+  # hypothesis-pytest is not needed
+  patches = [ ./hypothesis_pytest.patch ];
```

Comment:
I'd prefer doing this in `postPatch` using `substituteInPlace`.
Did you submit a patch upstream?

Reply:
I opened a PR, but I actually realized that it's building without the patch.